README.rst

```diff
@@ -66,5 +66,8 @@ a long list of results.
 .. |build| image:: https://travis-ci.org/ryan-roemer/django-cloud-browser.svg
    :target: https://travis-ci.org/ryan-roemer/django-cloud-browser
 
+.. |install| image:: https://dev.azure.com/michaelsethperel/django-cloud-browser/_apis/build/status/michaelperel.django-cloud-browser?branchName=master
```

Comment:
Consider renaming `|build|` to `|travis|` and `|install|` to `|azdo|` (or similar) to make naming more explicit.